scripts/check_doc_links.py

```diff
@@ -43,6 +43,15 @@
     "https://colab.research.google.com": 403,
 }
 
+IGNORE = {
+    "https://pypi.org/project/aea-crypto-fetchai/",
+    "https://pypi.org/project/aea-crypto-ethereum/",
+    "https://pypi.org/project/aea-crypto-cosmos/",
+    "https://github.com/fetchai/agents-aea/tree/main/plugins/aea-crypto-fetchai",
+    "https://github.com/fetchai/agents-aea/tree/main/plugins/aea-crypto-ethereum",
+    "https://github.com/fetchai/agents-aea/tree/main/plugins/aea-crypto-cosmos",
+}
+
```

Comment:
temporary. It can be removed after the next release.

Reply:
Please open issue so we don't forget